<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MentalHealth</class>
 <widget class="QWidget" name="MentalHealth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}
</string>
  </property>
  <widget class="QLabel" name="Mentalhealtlabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; font-style:italic;&quot;&gt;Mental Health Tracker &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DescriptionFixed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; font-style:italic;&quot;&gt;Hey, how are you feeling today? Let us know, we'll suggest you something based on that. Have a great day buddy!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="HappyButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Happy </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="SadButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sad </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="TiredButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tired</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="WorriedButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Worried </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ProductiveButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Productive</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogMoodButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Mood Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BacktoTabsfromMental">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Tabs!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
